### ISSUE TITLE
Suppress IP for all alerts

### ISSUE THREAD

**Issue by FaikCoskun** [snort3/snort3#137]
Is there any function that can suppress an ip address for all sids?
Works fine for one sid but for ex: there is an ip adresse that has traffic more than 20 sids I have to write all sid by_src and by_dst to suppress traffic because this traffic is valid for me.

Is there any other way to suppress all traffic/sids for one ip or am I missing the point ?

Thanks

**Comment by snortadmin** [comment]
Yes, redirect to the snort-users list and I'll explain so everyone sees it.  This is for development.

https://snort.org/community#mailing_lists

Russ
